src/Service/SF1500/SF1500.php

```diff
@@ -420,58 +396,135 @@ public function getEnhedNavnOgOverordnetOrganisationsId($organisationEnhedsId, $
     }
 
     /**
-     * Fetches bruger and organisation funktions id for nearest manager or null if no manager exists.
+     * Fetches bruger and organisation funktions id for managers from user id.
+     * Returns null if no manager exists.
      */
-    public function getNearestManagerBrugerAndFunktionsId($userId, $managerFunktionsTypeId)
+    public function getManagerBrugerAndFunktionsIdFromUserId($userId, $managerFunktionsTypeId)
     {
         $token = $this->fetchSAMLToken();
 
-        if (null === $token) {
-            return '';
-        }
-
         $organisationFunktionsId = $this->getOrganisationFunktionerFromUserId($userId);
 
-        // Select just one of the organisation funktioner(ansættelser).
-        if (is_array($organisationFunktionsId)) {
-            $organisationFunktionsId = reset($organisationFunktionsId);
+        if (null === $organisationFunktionsId) {
+            throw new SF1500Exception(sprintf('Could not find any organisation funktioner for user %s', $userId));
         }
 
-        $orgId = $this->getOrganisationEnhed($organisationFunktionsId, true);
+        $managers = [];
+        if (is_array($organisationFunktionsId)) {
```

Comment:
I think it should always be an array. It's weird that a function can return either an array or something else.

Reply:
Modified to always return array. Empty if none found.